openwisp_users/tests/test_admin.py

```diff
@@ -256,3 +256,24 @@ def test_admin_add_superuser_org_not_required(self):
         user = queryset.first()
         self.assertTrue(user.is_superuser)
         self.assertTrue(user.is_staff)
+
+    # def test_operator_change_user_permissions(self):
+    #     operator = self._create_operator()
+    #     operator.is_staff = True
+    #     # print(operator._meta.get_fields())
+    #     operator.user_permissions.set(Permission.objects.all())
```

Comment:
not sure if this is needed, leave it here for the moment but once you get the test to work you can try removing it and if the test doesn't fail you can remove it